.github/workflows/release-docker.yml

```diff
@@ -3,27 +3,31 @@ name: Docker
 on:
   push:
     branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
+    paths:
+      - ".github/workflows/release-docker.yml"
+      - "dockers/release/*"
```

Comment:
Do you want to add this to checkgroup now that it runs on pull request?

Reply:
Maybe worse to do it... Thx